File: onebot/src/main/java/cn/evole/onebot/sdk/response/misc/BooleanResp.java
package cn.evole.onebot.sdk.response.misc;

import com.google.gson.annotations.SerializedName;
import lombok.AllArgsConstructor;
import lombok.Data;

/**
 * Created on 2022/7/8.
 *
 * @author cnlimiter
 */
@Data
@AllArgsConstructor
public class BooleanResp {

    @SerializedName("yes")
    public boolean yes;

}

File: onebot/src/main/kotlin/sdk/response/misc/BooleanResp.kt
package cn.evolvefield.onebot.sdk.response.misc

import com.google.gson.annotations.SerializedName
import lombok.AllArgsConstructor
import lombok.Data

@Data
@AllArgsConstructor
class BooleanResp {
    @SerializedName("yes")
    var yes = false
}
